src/entity/field-names.js

```diff
@@ -0,0 +1,245 @@
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const bizTransaction = {
+  type: 'type',
+  bizTransaction: 'bizTransaction',
+};
+
+/**
+ * An object containing all the possible field names for a destinationElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const destination = {
+  type: 'type',
+  destination: 'destination',
+};
+
+/**
+ * An object containing all the possible field names for an errorDeclaration
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const errorDeclaration = {
+  declarationTime: 'declarationTime',
+  reason: 'reason',
+  correctiveEventIDs: 'correctiveEventIDs',
+};
+
+/**
+ * An object containing all the possible field names for an ilmd
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const ilmd = {
+  type: 'type',
+  format: 'format',
+};
+
+/**
+ * An object containing all the possible field names for persistentDisposition
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const persistentDisposition = {
+  set: 'set',
+  unset: 'unset',
+};
+
+/**
+ * An object containing all the possible field names for a quantityElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const quantityElement = {
+  epcClass: 'epcClass',
+  quantity: 'quantity',
+  uom: 'uom',
+};
+/**
+ * An object containing all the possible field names for a readPoint
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const readPoint = {
+  id: 'id',
+};
+/**
+ * An object containing all the possible field names for a sourceElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const source = {
+  type: 'type',
+  source: 'source',
+};
+/**
+ * An object containing all the possible field names for a vocabulary
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const vocabulary = {
+  type: 'type',
+  vocabularyElementList: 'vocabularyElementList',
+};
+/**
+ * An object containing all the possible field names for a vocabularyElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const vocabularyElement = {
+  id: 'id',
+  attributes: 'attributes',
+  children: 'children',
+};
+
+/**
+ * An object containing all the possible field names for a sensorElement
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const sensorElement = {
+  sensorMetadata: 'sensorMetadata',
+  sensorReport: 'sensorReport',
+};
+/**
+ * An object containing all the possible field names for a sensorMetadata
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const sensorMetadata = {
+  time: 'time',
+  startTime: 'startTime',
+  endTime: 'endTime',
+  deviceID: 'deviceID',
+  deviceMetadata: 'deviceMetadata',
+  rawData: 'rawData',
+  dataProcessingMethod: 'dataProcessingMethod',
+  bizRules: 'bizRules',
+};
+/**
+ * An object containing all the possible field names for a sensorReport
+ * More info here: https://www.gs1.org/standards/epcis
+ */
+const sensorReport = {
```

Comment:
did you take the file from thng-epcis as it was or did you update it? I don't see the exception field that has been added recently for example